src/scss/colorways.scss

```
@@ -575,6 +575,34 @@
   // placeholder
 }
 
+.gmk-fro-dot-yo-key, .gmk-fro-dot-yo-mod {
```

Apparently the CI doesn't like this statement as one line.

```suggestion
.gmk-fro-dot-yo-key,
.gmk-fro-dot-yo-mod {
```